### ISSUE TITLE
Build failure with http-client 0.5

### ISSUE THREAD

**Issue by snoyberg** [alphaHeavy/consul-haskell#14]
```
> /tmp/stackage-build8$ stack unpack consul-haskell-0.3
Unpacked consul-haskell-0.3 to /tmp/stackage-build8/consul-haskell-0.3/
> /tmp/stackage-build8/consul-haskell-0.3$ runghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup configure --package-db=clear --package-db=global --package-db=/var/stackage/work/builds/nightly/pkgdb --libdir=/var/stackage/work/builds/nightly/lib --bindir=/var/stackage/work/builds/nightly/bin --datadir=/var/stackage/work/builds/nightly/share --libexecdir=/var/stackage/work/builds/nightly/libexec --sysconfdir=/var/stackage/work/builds/nightly/etc --docdir=/var/stackage/work/builds/nightly/doc/consul-haskell-0.3 --htmldir=/var/stackage/work/builds/nightly/doc/consul-haskell-0.3 --haddockdir=/var/stackage/work/builds/nightly/doc/consul-haskell-0.3 --flags=
Configuring consul-haskell-0.3...
> /tmp/stackage-build8/consul-haskell-0.3$ runghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup build
Building consul-haskell-0.3...
Preprocessing library consul-haskell-0.3...

src/Network/Consul/Types.hs:30:0: error:
     warning: extra tokens at end of #ifdef directive
     #ifdef __GLASGOW_HASKELL__ <710
     ^
[1 of 3] Compiling Network.Consul.Types ( src/Network/Consul/Types.hs, dist/build/Network/Consul/Types.o )

src/Network/Consul/Types.hs:31:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()

src/Network/Consul/Types.hs:32:1: warning: [-Wunused-imports]
    The import of ‘Data.Traversable’ is redundant
      except perhaps to import instances from ‘Data.Traversable’
    To import instances alone, use: import Data.Traversable()
[2 of 3] Compiling Network.Consul.Internal ( src/Network/Consul/Internal.hs, dist/build/Network/Consul/Internal.o )

src/Network/Consul/Internal.hs:68:88: error:
    Not in scope: ‘checkStatus’

src/Network/Consul/Internal.hs:69:41: error:
    Not in scope: ‘checkStatus’
```

**Comment by sseveran** [comment]
I will update the package this weekend.

**Comment by sseveran** [comment]
I have released 0.4 to hackage. Let me know if I need to do anything else to update stackage.

**Comment by snoyberg** [comment]
Thanks, I'll add wai-middleware-consul back to Stackage :)
